pipelines/load-data-in-dk5/Jenkinsfile

```
@@ -38,15 +38,8 @@ pipeline {
             steps {
                 script {
                     if (currentBuild.resultIsBetterOrEqualTo('SUCCESS')) {
-                        def ARTY_SERVER = Artifactory.server 'arty'
-                        def ARTY_DOCKER = Artifactory.docker server: ARTY_SERVER, host: env.DOCKER_HOST
-                        def BUILD_INFO = Artifactory.newBuildInfo()
-                        BUILD_INFO.name = BUILD_NAME
-                        BUILD_INFO.env.capture = true
-                        BUILD_INFO.env.collect()
-                        BUILD_INFO = ARTY_DOCKER.push("$DOCKER_NAME", 'docker-ux', BUILD_INFO)
-                        ARTY_DOCKER.push("$DOCKER_NAME", 'docker-ux', BUILD_INFO)
-                        ARTY_SERVER.publishBuildInfo BUILD_INFO
+                        docker.image("${DOCKER_NAME}").push("${BUILD_NUMBER}")
```

Dette step og ovenstående step kan klares i en kommando - noget ala dette:

```suggestion
docker.withRegistry('https://docker-fbiscrum.artifacts.dbccloud.dk', 'DOCKER_LOGIN') {
  docker.build(
    "${DOCKER_NAME}", "-f pipelines/build-dk5-image/Dockerfile --no-cache ."
  ).push("${BUILD_NUMBER}")
}
```